batch/provision.yml

```diff
@@ -10,6 +10,9 @@
     - master_instance_type: m1.medium
     - core_instance_num: 1
     - core_instance_type: m1.medium
+    - tags:
+        Key: 'deployment'
+        Value: "{{ instance }}"
```

Comment:
Where does this variable come from?

Reply:
It will be set in the emr-vars.yml configuration file that is in the config repo.

Reply:
Any chance we can call it `COMMON_DEPLOYMENT`, like elsewhere?